pkgs/development/libraries/totem-pl-parser/default.nix

```diff
@@ -16,8 +16,16 @@ stdenv.mkDerivation rec {
     };
   };
 
-  nativeBuildInputs = [ meson ninja pkg-config gettext gobject-introspection ];
-  buildInputs = [ libxml2 ];
+  strictDeps = true;
+  nativeBuildInputs = [ meson ninja pkg-config gettext glib ];
+  buildInputs = [ libxml2 glib ]
```

Comment:
Can you explain why we suddenly need glib?

Reply:
I think this always depended on glib, only it took it from something that propagated it.

Reply:
gobject-introspection propagates it.